src/js/index.js

```diff
@@ -443,7 +443,7 @@ class Router {
   }
 }
 
-if (CONTEXT === "production" && GOOGLE_TAG_MANAGER_ID !== "undefined") {
+if (CONTEXT === "production" || GOOGLE_TAG_MANAGER_ID !== "undefined") {
```

Comment:
Shouldn't this stay as a logical AND operator? If it's in production but the GTM ID is not set it wouldn't work, would it?

Reply:
I had to change in order to be able to test it at all locally unless I did production builds. Also, my general thinking was that, in production, it is very unlikely that the GTM-ID is not set so maybe it is ok? I am happy to change it back to an `&&` though. Let me know.

Reply:
I understand for dev it's necessary but I don't think we should commit this change. This has two implications:
1. It will try to initialise in prod even if there is no ID set (not so bad, since we'll probably get an error to fix it), but
2. If we accidentally add an ID to staging, sandbox or any other environment it will initialise and we might not notice this, causing nonsense data to be included into prod data

Reply:
Actually, I thought about it and I was silly 😄  one can just set the `NODE_ENV` to "production" locally and it should work just as well. I will change back to `&&`